wrappers/swift/SDWLogManager.swift

```diff
@@ -0,0 +1,182 @@
+
+    - Parameters:
+        - tenantToken: A `String` that contains the tenant token.
+        - config: A `Dictionary` that contains a custom configuration.
+
+    - Returns:
+        - A `Logger` instance.
+    */
+    public static func initForTenant(tenantToken: String!, config: [AnyHashable: Any]? = nil) -> Logger? {
+        if let odwLogger = ODWLogManager.initForTenant(tenantToken, withConfig: config) {
+            return Logger(logger: odwLogger)
+        } else {
+            return nil
+        }
+    }
+
+    /**
+    Retrieves a new instance of `Logger` for logging telemetry events. Initializes the telemetry logging system,
+    if not already initialized, with the default configuration using the specified tenant token and source.
+
+    - Parameters:
+        - tenantToken: A `String` that contains the tenant token.
+        - withSource: A `String` that contains the name of the source of events.
+
+    - Returns:
+        - A `Logger` instance that points to the logger for the specified tenant token and source.
+    */
+    public static func loggerWithTenant(tenantToken: String!, withSource source: String! = "") -> Logger? {
+        if let odwLogger = ODWLogManager.logger(withTenant: tenantToken, source: source) {
+            return Logger(logger: odwLogger)
+        } else {
+            return nil
+        }
+    }
+
+    /**
+    Retrieves a new instance of `Logger` for logging telemetry events.
+    This method expects that the logging system has already been initialized.
+
+    - Parameters:
+        - tenantToken: A `String` that contains the tenant token.
+        - withSource: A `String` that contains the name of the source of events.
+        - config: A `Dictionary` that contains a custom configuration.
+
+    - Returns:
+        - A `Logger` instance that points to the logger for the specified tenant token and source.
+    */
+    public static func loggerWithTenant(tenantToken: String!, withSource source: String!, withConfig config: ODWLogConfiguration!) -> Logger?{
+        if let odwLogger = ODWLogManager.logger(withTenant: tenantToken, source: source, withConfig: config) {
+            return Logger(logger: odwLogger)
+        } else {
+            return nil
+        }
+    }
+
+    /**
+    Retrieves a new instance of `Logger` for logging telemetry events.
+
+    - Note:
+        It requires to previously call `loggerWithTenant` method.
+
+    - Parameters:
+        - source: A `String` that contains the name of the source of events sent by this logger instance.
+
+    - Return:
+        - A `Logger` instance that points to the logger for source.
+    */
+    public static func loggerForSource(source: String!) -> Logger? {
+        if let odwLogger = ODWLogManager.logger(forSource: source) {
+            return Logger(logger: odwLogger)
+        } else {
+            return nil
+        }
+    }
+
+    /// Attempts to send any pending telemetry events that are currently cached either in memory, or on disk.
```

Comment:
Functions above have documentation in the form of /*** .. */ while this is just // . Please standardize.

Reply:
I found two ways of adding the comments to the functions as per the conventions on line.
/** */ and ///.
/// I have used only places where comments are concise and don't need the parameters and returns specified separately, rest of the places /** */ is being used.

Reply:
Though i just checked cpp and they seemed to be used ///. So I think we can change it to /// throughout for the consistency.
As that is just about the comments formatting, will take that as separate change.